### ISSUE TITLE
Empty values (e.g. `""`, `0`) are being formatted incorrectly

### ISSUE THREAD

**Issue by louis123562** [NickKaramoff/pretty-money#8]
hey,

is it possible to format empty values to an exptected output?

Like 

```
const prettyEuro = {
   currency: "€",
   position: "after",
   spaced: true,
   decimals: "fixed",
   maxDecimals: 2,
   thousandsDelimiter: ".",
   decimalDelimiter: ","
};   

console.log( prettyMoney(prettyEuro, "0");
```
to 

`0,00 € `

and not

` ,0 € `

Thanks in advance, Louis

**Comment by kytta** [comment]
Yup, it's a bug; probably has to do with `0` being false-y

**Comment by kytta** [comment]
Fixed in [v1.1.1](https://github.com/NickKaramoff/pretty-money/releases/tag/v1.1.1) 🎉

**Comment by louis123562** [comment]
Will test this evening - thanks a lot :)
